elyra/pipeline/processor_kfp.py

```diff
@@ -91,20 +91,24 @@ def export(self, pipeline, pipeline_export_format, pipeline_export_path, overwri
 
         pipeline_name = pipeline.name
 
+        # Since pipeline_export_path may be relative to the notebook directory, ensure
+        # we're using its absolute form.
+        export_path = self.get_absolute_path(pipeline_export_path)
+
```

Comment:
could you please add `absolute` to the variable name to make it easier to read? How about `absolute_pipeline_export_path` if that is not too long... witch associates with the `pipeline_export_path` and hint that it's now `absolute`

Reply:
Yes, I totally agree.